lib/loc/make.go

```diff
@@ -48,6 +58,5 @@ func MakeLocator(app string) (*Locator, error) {
 		}
 		return NewLocator(defaults.SystemAccountOrg, parts[0], version)
 	}
-	return nil, trace.BadParameter(
-		"invalid app name format: %v, should be: 'repo/name:ver' or 'name:ver' or 'name'", app)
+	return nil, trace.BadParameter("invalid app name format: %v, should be: 'repo/name:ver' or 'name:ver' or 'name'", app)
```

Comment:
it should be `package` instead of `app` and I would place the placeholder in quotes to emphasize the possibly empty input.
Also, why use truncated words in an error message? `repository/name:version`